README.md

```diff
@@ -1,7 +1,120 @@
-# Tour Savvy App
+# Tour Savvy App Backend
+<a name="readme-top"></a>
 
+# 📗 Table of Contents
 
-## Authors:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Api documentation](#api-doc)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Tour_Savvy_Backend] <a name="about-project"></a>
+
+Tour_Savvy is a rails api app that allows operations like get, delete, post for packages and reservations. It also supports authentication for user login and signup.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <ul>
+    <li><a href="https://www.postgresql.org/">Postgresql v15.3</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails v7.1.1</a></li>
+    <li><a href="https://rspec.info/">Rspec v3.12</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Index and Show Packages**
+- **Index and Show Reservations**
+- **Add/Remove Reservation**
+- **Add/Remove Package**
+- **Authentication and authorization**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: 
+
+```sh
+ [Ruby] (https://rubyinstaller.org/downloads/)
+ Rails: gem install rails
+```
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/zuhebahmed88091/tour_savvy_backend.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd tour_savvy_backend
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+  configure database.yml in config to user your postgres database
+  rails db:seed (this is to seed a admin in database)
+```
+
+### Run tests
+
+To run tests, run the following command: 
+
+Example command:
+
+```sh
+  rspec ./spec/models
+```
```

Comment:
- [ ] This command is not going to work:

![image](https://github.com/zuhebahmed88091/tour_savvy_backend/assets/57421823/abddb684-3df6-4e50-9dff-d500da9a88e3)

This is because that folder does not exist.  Kindly, include a command to correctly run your tests. 👍

Reply:
The dev branch was not pulled correctly which had the test files. Now we have corrected the issue it should work now.
Thanks for the feedback. 👍😃 
![rspec](https://github.com/zuhebahmed88091/tour_savvy_backend/assets/117600135/fc86aa7e-e2ae-43cb-a586-5cf84d9fec11)